gnd/src/main/java/com/google/android/gnd/system/AuthenticationManager.java

```diff
@@ -184,7 +184,7 @@ public User getUser() {
 
   public static class User {
 
-    public static final User ANONYMOUS = new User("", "", "");
+    public static final User ANONYMOUS = new User("", "", "Unknown user");
```

Comment:
All user-visible strings should come from translatable resources, so this string should ideally be decided at rendering time.

Reply:
Done